run_tests.sh

```diff
@@ -1,3 +1,3 @@
 carthage bootstrap --platform iOS --cache-builds;
 xcrun simctl uninstall booted com.duckduckgo.mobile.ios;
-xcodebuild test -project DuckDuckGo.xcodeproj -scheme DuckDuckGo CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 6s';
+xcodebuild test -quiet -project DuckDuckGo.xcodeproj -scheme DuckDuckGo CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator,name=iPhone 6s';
```

Comment:
Added quiet other wise we hit our 4mb log limit on travis when both tests scripts are run.